middleware/errorHandler.js

```diff
@@ -0,0 +1,44 @@
+const { constants } = require("../constants");
+const errorHandler=(err, req, res, next)=>{
```

Comment:
error handling should be done based on the error message instead of response code ..to avoid sending response again after sending the response in controller